src/Map/src/Bridge/Google/src/Option/ControlPosition.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Bridge\Google\Option;
+
+/**
+ * Identifiers used to specify the placement of controls on the map.
+ * Controls are positioned relative to other controls in the same layout position.
+ * Controls that are added first are positioned closer to the edge of the map.
+ * Usage of "logical values" (see https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) is recommended
+ * in order to be able to automatically support both left-to-right (LTR) and right-to-left (RTL) layout contexts.
+ *
+ * @see https://developers.google.com/maps/documentation/javascript/reference/control#ControlPosition
+ *
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+enum ControlPosition: int
+{
+    /**
+     * Equivalent to {@see self::BOTTOM_CENTER} in both LTR and RTL.
+     */
+    case BLOCK_END_INLINE_CENTER = 24;
```

Comment:
The symfony code style say "PascalCase" for the enums ...  don't know what to do there 

(you do use PascalCase elsewhere right?)

Reply:
I simply re-used the same case than GoogleMaps API, but this can modified for sure.

Reply:
I'd follow the recommendation and  use only logical i think. That reduces the number of choices / questions, and easier to document/explain... but that's just my feeling :)